### ISSUE TITLE
Fix token

### ISSUE THREAD

**Issue by paulhobbel** [ItsDizzy/angular2-auth#13]
Like you can see in my small documentation on the README file, I am using a weird work around to check if a token has an actually token set.
```ts
  loggedIn() {
    let token = this._tokenService.getToken();
    
    if(token && token.token) {
      return !token.isExpired();
    }
    return false;
  }
```
This should be done better, if no token is set the token should be null, not token.token

**Comment by nimatrazmjo** [comment]
Cool

**Comment by paulhobbel** [comment]
This has been fixed in 9b11953b
